src/lib.rs

```diff
@@ -83,8 +84,8 @@ mod response;
 pub mod middleware;
 
 pub use http_types;
+pub use http_types::mime;
```

Comment:
We're not re-exporting this from tide either -- though perhaps we should? I'm curious about the reasoning here.